verilog/tools/ls/README.md

```diff
@@ -89,14 +89,69 @@ to get it started up on our Verilog/SystemVerilog projects.
 
 ### Neovim
 
-Make sure to have version `0.5.0` or newer and install the [nvim-lpsconfig](https://github.com/neovim/nvim-lspconfig/) plugin.
-Enable the verible config:
+Make sure to have version `0.5.0` or newer and install the [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig/) plugin.
```

Comment:
The `verible-verilog-ls` binary to call is never mentioned in this configuration. Is it because neovim has already some preconfiguration for Verible ? If so, we should mention it here. It would help understand _why_ we want version `>= 0.5.0`.

Reply:
nvim-lspconfig is hosting the verible ls in neovim, it has a default cmd attribute for verible which can be modified if need be

Reply:
I don't really know where that version comes from but it might have something to do with the nvim-lspconfig plugin